src/table/__tests__/inline-editing-test-utils.test.tsx

```diff
@@ -16,24 +16,24 @@ const defaultItems: Item[] = [
   { id: 3, name: 'Bananas' },
 ];
 
-const editableColumns: TableProps.ColumnDefinition<Item, string | number>[] = [
+const editableColumns: TableProps.ColumnDefinition<Item>[] = [
   {
     header: 'id',
-    cell: (item: Item, { isEditing }) => {
-      if (isEditing) {
-        return <input type="number" value={item.id} readOnly={true} data-testid={`id-editing-${item.id}`} />;
-      }
-      return item.id;
-    },
     editConfig: {
       ariaLabel: 'Edit id',
       errorIconAriaLabel: 'Error',
       editIconAriaLabel: 'Edit',
     },
+    cell: (item, { isEditing }: TableProps.CellContext<number>) => {
+      if (isEditing) {
+        return <input type="number" value={item.id} readOnly={true} data-testid={`id-editing-${item.id}`} />;
+      }
+      return item.id;
+    },
   },
   {
     header: 'name',
-    cell: (item: Item, { isEditing }) => {
+    cell: (item: Item, { isEditing }: TableProps.CellContext<string>) => {
```

Comment:
Why is this explicit type needed? Is it possible to make it coming by default?

If needed, we can provide an empty `context` object for non-editable cells, if we need fixed arity between to modes

Reply:
Strictly speaking, the explicit type is not needed in this particular usage. I'll remove it.